<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Function reference â€¢ eIF4F.analysis</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link href="https://cdnjs.cloudflare.com/ajax/libs/bootswatch/3.4.0/flatly/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><meta property="og:title" content="Function reference"><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=UA-12345678-0"></script><script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-12345678-0');
</script></head><body data-spy="scroll" data-target="#toc">
    

    <div class="container template-reference-index">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">eIF4F.analysis</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">0.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li>
      <a href="../articles/eIF4F_analysis.html">eIF4F_analysis</a>
    </li>
  </ul></li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/a3609640/eIF4F.analysis/" class="external-link">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">
    <div class="page-header">
      <h1>Reference</h1>
    </div>

    <table class="ref-index"><colgroup><col class="alias"><col class="title"></colgroup><tbody><tr><th colspan="2">
          <h2 id="directory-initialization">Directory initialization <a href="#directory-initialization" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="initialize_dir.html">initialize_dir()</a></code> </p>
        </td>
        <td><p>Set output directories</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="format-initialization">Format initialization <a href="#format-initialization" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="initialize_format.html">initialize_format()</a></code> </p>
        </td>
        <td><p>Set format for font and color of plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="data-initialization">Data initialization <a href="#data-initialization" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>reads datasets from the download data files</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="initialize_data.html">initialize_data()</a></code> </p>
        </td>
        <td><p>A wrapper function reads datasets from the download data files</p></td>
      </tr><tr><td>
          <p><code><a href="initialize_cnv_data.html">initialize_cnv_data()</a></code> </p>
        </td>
        <td><p>Read all CNV related datasets from TCGA</p></td>
      </tr><tr><td>
          <p><code><a href="initialize_RNAseq_data.html">initialize_RNAseq_data()</a></code> </p>
        </td>
        <td><p>Read all RNA-seq related datasets from TCGA and GTEx</p></td>
      </tr><tr><td>
          <p><code><a href="initialize_survival_data.html">initialize_survival_data()</a></code> </p>
        </td>
        <td><p>Read RNA-seq and survival datasets from TCGA</p></td>
      </tr><tr><td>
          <p><code><a href="initialize_proteomics_data.html">initialize_proteomics_data()</a></code> </p>
        </td>
        <td><p>Read all proteomics related datasets from CCLE and CPTAC LUAD</p></td>
      </tr><tr><td>
          <p><code><a href="initialize_phosphoproteomics_data.html">initialize_phosphoproteomics_data()</a></code> </p>
        </td>
        <td><p>Read all phosphoproteomics related datasets from CPTAC LUAD</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-processing-cnv-related-datasets">Internal functions for processing CNV related datasets <a href="#internal-functions-for-processing-cnv-related-datasets" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-get_TCGA_CNV.html">.get_TCGA_CNV()</a></code> </p>
        </td>
        <td><p>Read CNV threshold dataset from TCGA</p></td>
      </tr><tr><td>
          <p><code><a href="dot-get_TCGA_CNV_value.html">.get_TCGA_CNV_value()</a></code> </p>
        </td>
        <td><p>Read CNV value dataset from TCGA</p></td>
      </tr><tr><td>
          <p><code><a href="dot-get_TCGA_CNV_ratio.html">.get_TCGA_CNV_ratio()</a></code> </p>
        </td>
        <td><p>Read CNV ratio dataset from TCGA</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-processing-deg-related-datasets">Internal functions for processing DEG related datasets <a href="#internal-functions-for-processing-deg-related-datasets" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-get_TCGA_GTEX_RNAseq.html">.get_TCGA_GTEX_RNAseq()</a></code> </p>
        </td>
        <td><p>Read recomputed RNAseq data from both TCGA and GTEx</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-processing-survival-related-datasets">Internal functions for processing survival related datasets <a href="#internal-functions-for-processing-survival-related-datasets" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-get_TCGA_RNAseq.html">.get_TCGA_RNAseq()</a></code> </p>
        </td>
        <td><p>Read the RNAseq data from TCGA</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="cnv-analysis">CNV analysis <a href="#cnv-analysis" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>CNV analyses of EIF genes in TCGA data</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="EIF4F_CNV_analysis.html">EIF4F_CNV_analysis()</a></code> </p>
        </td>
        <td><p>Perform all CNV related analysis and generate plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-composite-functions-to-call-cnv-data-analysis-and-plotting">Internal composite functions to call CNV data analysis and plotting <a href="#internal-composite-functions-to-call-cnv-data-analysis-and-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-plot_bargraph_CNV_TCGA.html">.plot_bargraph_CNV_TCGA()</a></code> </p>
        </td>
        <td><p>Summary of CNV statuses in bar plots</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_matrix_CNVcorr_TCGA.html">.plot_matrix_CNVcorr_TCGA()</a></code> </p>
        </td>
        <td><p>Correlation matrix for CNV values</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_boxgraph_CNVratio_TCGA.html">.plot_boxgraph_CNVratio_TCGA()</a></code> </p>
        </td>
        <td><p>CNV ratios in tumors vs adjacent normal tissues across tumor types</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-cnv-data-analysis">Internal functions for CNV data analysis <a href="#internal-functions-for-cnv-data-analysis" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-CNV_all_cancer.html">.CNV_all_cancer()</a></code> </p>
        </td>
        <td><p>Calculates the frequency of CNV status in all TCGA cancer types combined</p></td>
      </tr><tr><td>
          <p><code><a href="dot-CNV_ind_cancer.html">.CNV_ind_cancer()</a></code> </p>
        </td>
        <td><p>Calculates the frequency of CNV status in individual TCGA cancer types</p></td>
      </tr><tr><td>
          <p><code><a href="dot-CNVratio_tumor.html">.CNVratio_tumor()</a></code> </p>
        </td>
        <td><p>Calculates the frequency of CNV status in all TCGA cancer types combined</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-plotting">Internal functions for plotting <a href="#internal-functions-for-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-CNV_sum_barplot.html">.CNV_sum_barplot()</a></code> </p>
        </td>
        <td><p>Stacked bar plots of the CNV status summary</p></td>
      </tr><tr><td>
          <p><code><a href="dot-CNV_barplot.html">.CNV_barplot()</a></code> </p>
        </td>
        <td><p>Stacked bar plots of the CNV status</p></td>
      </tr><tr><td>
          <p><code><a href="dot-matrix_plot.html">.matrix_plot()</a></code> </p>
        </td>
        <td><p>Correlation matrix for gene pairs</p></td>
      </tr><tr><td>
          <p><code><a href="dot-CNVratio_boxplot.html">.CNVratio_boxplot()</a></code> </p>
        </td>
        <td><p>Box plots of the CNV ratios in tumors vs adjacent normals</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="deg-analysis">DEG analysis <a href="#deg-analysis" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Differential gene expression (ratio) analysis of EIF4F genes in TCGA</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="EIF4F_DEG_analysis.html">EIF4F_DEG_analysis()</a></code> </p>
        </td>
        <td><p>Perform differential gene expression or ratio analysis and
generate plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-composite-functions-to-call-deg-gene-analysis-and-plotting">Internal composite functions to call DEG gene analysis and plotting <a href="#internal-composite-functions-to-call-deg-gene-analysis-and-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-plot_boxgraph_RNAseq_TCGA.html">.plot_boxgraph_RNAseq_TCGA()</a></code> </p>
        </td>
        <td><p>Compare the expression of EIF4F genes</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_boxgraph_RNAratio_TCGA.html">.plot_boxgraph_RNAratio_TCGA()</a></code> </p>
        </td>
        <td><p>Compare the RNA ratios between EIF4F genes</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-differential-expression-analysis">Internal functions for differential expression analysis <a href="#internal-functions-for-differential-expression-analysis" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-RNAseq_all_gene.html">.RNAseq_all_gene()</a></code> </p>
        </td>
        <td><p>Compares expressions among genes in tumor samples</p></td>
      </tr><tr><td>
          <p><code><a href="dot-RNAseq_ind_gene.html">.RNAseq_ind_gene()</a></code> </p>
        </td>
        <td><p>Analyzes differential gene expression in tumors vs adjacent normal tissues</p></td>
      </tr><tr><td>
          <p><code><a href="dot-RNAseq_tumortype.html">.RNAseq_tumortype()</a></code> </p>
        </td>
        <td><p>Select gene expression data in primary, metastatic tumors vs adjacent normal
tissues from all TCGA cancer types combined</p></td>
      </tr><tr><td>
          <p><code><a href="dot-RNAratio_calculation.html">.RNAratio_calculation()</a></code> </p>
        </td>
        <td><p>Calculate RNA ratios</p></td>
      </tr><tr><td>
          <p><code><a href="dot-RNAratio_selection.html">.RNAratio_selection()</a></code> </p>
        </td>
        <td><p>Select RNA ratio data for plotting</p></td>
      </tr><tr><td>
          <p><code><a href="dot-RNAratio_tumortype.html">.RNAratio_tumortype()</a></code> </p>
        </td>
        <td><p>Analyzes differential RNA ratios in primary, metastatic tumors vs
adjacent normal tissues from all TCGA cancer types combined</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-differential-expression-analysis-plotting">Internal functions for differential expression analysis plotting <a href="#internal-functions-for-differential-expression-analysis-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-RNAseq_grouped_boxplot.html">.RNAseq_grouped_boxplot()</a></code> </p>
        </td>
        <td><p>Grouped box plots of RNA expression across tumors</p></td>
      </tr><tr><td>
          <p><code><a href="dot-RNAseq_boxplot.html">.RNAseq_boxplot()</a></code> </p>
        </td>
        <td><p>Box plots of differential gene expression across tumors</p></td>
      </tr><tr><td>
          <p><code><a href="dot-violinplot.html">.violinplot()</a></code> </p>
        </td>
        <td><p>Violin plots of differential gene expression/ratio in primary,
metastatic tumors vs adjacent normal tissues</p></td>
      </tr><tr><td>
          <p><code><a href="dot-RNAratio_boxplot.html">.RNAratio_boxplot()</a></code> </p>
        </td>
        <td><p>Box plots of differential RNA ratios across tumors</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="survival-analysis">Survival analysis <a href="#survival-analysis" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Survival analyses on EIF4F gene expression in TCGA tumors</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="EIF4F_Survival_analysis.html">EIF4F_Survival_analysis()</a></code> </p>
        </td>
        <td><p>Perform all related survival analysis and generate plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-composite-functions-to-call-survival-analysis-and-plotting">Internal composite functions to call survival analysis and plotting <a href="#internal-composite-functions-to-call-survival-analysis-and-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-plot_KM_RNAseq_TCGA.html">.plot_KM_RNAseq_TCGA()</a></code> </p>
        </td>
        <td><p>Survival analyses of TCGA patients with expression in their tumors by
Kaplan Meier method</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_CoxPH_RNAseq_TCGA.html">.plot_CoxPH_RNAseq_TCGA()</a></code> </p>
        </td>
        <td><p>Survival analyses of TCGA patients with expression in their tumors by
Cox-PH method</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-survival-analysis">Internal functions for survival analysis <a href="#internal-functions-for-survival-analysis" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-KM_curve.html">.KM_curve()</a></code> </p>
        </td>
        <td><p>Kaplan Meier survival analyses of gene expression</p></td>
      </tr><tr><td>
          <p><code><a href="dot-univariable_analysis.html">.univariable_analysis()</a></code> </p>
        </td>
        <td><p>Univariable Cox-PH analyses of gene expression</p></td>
      </tr><tr><td>
          <p><code><a href="dot-multivariable_analysis.html">.multivariable_analysis()</a></code> </p>
        </td>
        <td><p>Multivariable Cox-PH analyses of gene expression</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-survival-analysis-plotting">Internal functions for survival analysis plotting <a href="#internal-functions-for-survival-analysis-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-forest_graph.html">.forest_graph()</a></code> </p>
        </td>
        <td><p>Forest plots of COX-PH results</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="pca-analysis">PCA analysis <a href="#pca-analysis" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>PCA on EIF4F expression in GTEx, or/and TCGA studies</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="EIF4F_PCA.html">EIF4F_PCA()</a></code> </p>
        </td>
        <td><p>Perform PCA and generate plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-composite-functions-to-call-call-pca-and-plot-results">Internal composite functions to call call PCA and plot results <a href="#internal-composite-functions-to-call-call-pca-and-plot-results" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-plot_PCA_TCGA_GTEX.html">.plot_PCA_TCGA_GTEX()</a></code> </p>
        </td>
        <td><p>Perform PCA on RNAseq data from all tumors and healthy tissues</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_PCA_TCGA_GTEX_tumor.html">.plot_PCA_TCGA_GTEX_tumor()</a></code> </p>
        </td>
        <td><p>Perform PCA on RNAseq data from one tumor type and its matched healthy
tissue</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_PCA_CPTAC_LUAD.html">.plot_PCA_CPTAC_LUAD()</a></code> </p>
        </td>
        <td><p>Perform PCA on proteomics data from CPATC LUADs and matched healthy
lung tissues</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-to-select-rna-seq-data-from-specific-study-groups-for-pca">Internal functions to select RNA-seq data from specific study groups for PCA <a href="#internal-functions-to-select-rna-seq-data-from-specific-study-groups-for-pca" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-get_df_subset.html">.get_df_subset()</a></code> </p>
        </td>
        <td><p>Select RNAseq data based on the sample types</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-to-perform-pca-or-imputed-pca">Internal functions to perform PCA or imputed PCA <a href="#internal-functions-to-perform-pca-or-imputed-pca" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-RNAseq_PCA.html">.RNAseq_PCA()</a></code> </p>
        </td>
        <td><p>Perform PCA on RNAseq data</p></td>
      </tr><tr><td>
          <p><code><a href="dot-protein_imputePCA.html">.protein_imputePCA()</a></code> </p>
        </td>
        <td><p>Perform imputPCA on proteomics data</p></td>
      </tr><tr><td>
          <p><code><a href="dot-biplot_title.html">.biplot_title()</a></code> </p>
        </td>
        <td><p>Generate title for PCA plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-to-plot-the-pca-results">Internal functions to plot the PCA results <a href="#internal-functions-to-plot-the-pca-results" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-biplot.html">.biplot()</a></code> </p>
        </td>
        <td><p>Plot PCA results as biplot, scree and matrix plots</p></td>
      </tr><tr><td>
          <p><code><a href="dot-selected_biplot.html">.selected_biplot()</a></code> </p>
        </td>
        <td><p>Plot subgroups of PCA results as biplots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="cor-analysis">COR analysis <a href="#cor-analysis" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Analyses on EIF4F correlating genes (CORs)</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="EIF4F_Corrgene_analysis.html">EIF4F_Corrgene_analysis()</a></code> </p>
        </td>
        <td><p>Perform differential correlation analyses and generate plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-composite-functions-to-analyze-the-eif-f-cors">Internal composite functions to analyze the EIF4F CORs <a href="#internal-composite-functions-to-analyze-the-eif-f-cors" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-plot_Corr_RNAseq_TCGA_GTEX.html">.plot_Corr_RNAseq_TCGA_GTEX()</a></code> </p>
        </td>
        <td><p>Perform correlation analysis on RNAseq data from all tumors and
healthy tissues</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-to-identify-correlating-genes-for-eif-f-genes">Internal functions to identify correlating genes for EIF4F genes <a href="#internal-functions-to-identify-correlating-genes-for-eif-f-genes" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-EIF_correlation.html">.EIF_correlation()</a></code> </p>
        </td>
        <td><p>Identify EIF4F correlating genes</p></td>
      </tr><tr><td>
          <p><code><a href="dot-correlation_coefficient.html">.correlation_coefficient()</a></code> </p>
        </td>
        <td><p>Calculate the correlation co-efficiency between two gene expressions</p></td>
      </tr><tr><td>
          <p><code><a href="dot-is_significant_poscor.html">.is_significant_poscor()</a></code> </p>
        </td>
        <td><p>Select positive correlating genes based on coefficiency and pvalue</p></td>
      </tr><tr><td>
          <p><code><a href="dot-is_significant_negcor.html">.is_significant_negcor()</a></code> </p>
        </td>
        <td><p>Select negative correlating genes based on coefficiency and pvalue</p></td>
      </tr><tr><td>
          <p><code><a href="dot-summarize_counts.html">.summarize_counts()</a></code> </p>
        </td>
        <td><p>Summary of total number of CORs identified for each EIF4F gene</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-for-correlation-analysis-and-plotting">Internal functions for correlation analysis and plotting <a href="#internal-functions-for-correlation-analysis-and-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-CORs_vennplot.html">.CORs_vennplot()</a></code> </p>
        </td>
        <td><p>Plot Venn diagrams for correlating genes</p></td>
      </tr><tr><td>
          <p><code><a href="dot-combine_CORs_summary.html">.combine_CORs_summary()</a></code> </p>
        </td>
        <td><p>Combine the summary of number of posCORs or negCORs from sample types</p></td>
      </tr><tr><td>
          <p><code><a href="dot-CORs_summary_bargraph.html">.CORs_summary_bargraph()</a></code> </p>
        </td>
        <td><p>Plot bargraph for the numbers of correlating genes</p></td>
      </tr><tr><td>
          <p><code><a href="dot-combine_CORs_list.html">.combine_CORs_list()</a></code> </p>
        </td>
        <td><p>Combine all the EIF4F correlating gene data from tumor and healthy
tissues</p></td>
      </tr><tr><td>
          <p><code><a href="dot-is_significant_correlation.html">.is_significant_correlation()</a></code> </p>
        </td>
        <td><p>Select both positive and negative correlating genes based on
coefficiency and pvalue</p></td>
      </tr><tr><td>
          <p><code><a href="dot-CORs_coeff_heatmap.html">.CORs_coeff_heatmap()</a></code> </p>
        </td>
        <td><p>Visualize associations between different sources of correlation data
and reveal potential pattern</p></td>
      </tr><tr><td>
          <p><code><a href="dot-get_cluster_genes.html">.get_cluster_genes()</a></code> </p>
        </td>
        <td><p>Retrieve the gene names from each cluster in heatmap</p></td>
      </tr><tr><td>
          <p><code><a href="dot-pathway_dotplot.html">.pathway_dotplot()</a></code> </p>
        </td>
        <td><p>Plot the enriched pathways from gene lists</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="rna-protein-correlation-analysis">RNA protein Correlation analysis <a href="#rna-protein-correlation-analysis" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Correlation between RNA and protein levels for EIF4F</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="EIF4F_RNA_pro_correlation.html">EIF4F_RNA_pro_correlation()</a></code> </p>
        </td>
        <td><p>Perform RNA protein correlation and generate scatter plots</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-composite-function-to-perform-rna-protein-correlation">Internal composite function to perform RNA protein correlation <a href="#internal-composite-function-to-perform-rna-protein-correlation" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-plot_scatter_RNApro_CCLE.html">.plot_scatter_RNApro_CCLE()</a></code> </p>
        </td>
        <td><p>Correlation between CCLE RNAseq and proteomics data</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_scatter_RNApro_LUAD.html">.plot_scatter_RNApro_LUAD()</a></code> </p>
        </td>
        <td><p>Correlation between LUAD RNAseq and proteomics data</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-to-select-eif-rna-pro-data">Internal functions to select EIF RNA/pro data <a href="#internal-functions-to-select-eif-rna-pro-data" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-get_CCLE_RNAseq_subset.html">.get_CCLE_RNAseq_subset()</a></code> </p>
        </td>
        <td><p>Select subset of CCLE RNAseq data</p></td>
      </tr><tr><td>
          <p><code><a href="dot-get_CCLE_Proteomics_subset.html">.get_CCLE_Proteomics_subset()</a></code> </p>
        </td>
        <td><p>Select subset of CCLE proteomics data</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-functions-to-plot-rna-protein-correlation">Internal functions to plot RNA protein correlation <a href="#internal-functions-to-plot-rna-protein-correlation" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-RNApro_scatterplot.html">.RNApro_scatterplot()</a></code> </p>
        </td>
        <td><p>Scatter plots of correlation between RNAseq and proteomics data</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h2 id="phoproteomics-and-phosphoproteomics-analysis">Phoproteomics and phosphoproteomics analysis <a href="#phoproteomics-and-phosphoproteomics-analysis" class="anchor" aria-hidden="true"></a></h2>
          <p class="section-desc"></p><p>Co-expression and differential expression analysis of EIF4F subunits</p>
        </th>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="exported-function">Exported function <a href="#exported-function" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="EIF4F_Proteomics_analysis.html">EIF4F_Proteomics_analysis()</a></code> </p>
        </td>
        <td><p>Analyze co-expression among EIF4F subunits and differential expression</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-composite-function-for-coexpression-and-differential-expression-analysis">Internal composite function for coexpression and differential expression analysis <a href="#internal-composite-function-for-coexpression-and-differential-expression-analysis" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-plot_scatterplot_protein_CPTAC.html">.plot_scatterplot_protein_CPTAC()</a></code> </p>
        </td>
        <td><p>protein-protein coexpression in CPTAC LUAD</p></td>
      </tr><tr><td>
          <p><code><a href="dot-plot_boxgraph_protein_CPTAC.html">.plot_boxgraph_protein_CPTAC()</a></code> </p>
        </td>
        <td><p>Comparison of protein and phosphorylation levels among different
stages of LUAD</p></td>
      </tr></tbody><tbody><tr><th colspan="2">
          <h3 id="internal-function-to-select-rna-and-protein-expression-data-and-plotting">Internal function to select RNA and protein expression data and plotting <a href="#internal-function-to-select-rna-and-protein-expression-data-and-plotting" class="anchor" aria-hidden="true"></a></h3>
          <p class="section-desc"></p>
        </th>
      </tr></tbody><tbody><tr><td>
          <p><code><a href="dot-protein_scatterplot.html">.protein_scatterplot()</a></code> </p>
        </td>
        <td><p>Scatter plots of protein coexpression</p></td>
      </tr><tr><td>
          <p><code><a href="dot-get_CPTAC_LUAD_Proteomics_subset.html">.get_CPTAC_LUAD_Proteomics_subset()</a></code> </p>
        </td>
        <td><p>Select subset of CPTAC proteomics data</p></td>
      </tr><tr><td>
          <p><code><a href="dot-get_CPTAC_LUAD_Phosproteomics_subset.html">.get_CPTAC_LUAD_Phosproteomics_subset()</a></code> </p>
        </td>
        <td><p>Select subset of CPTAC phosproteomics data</p></td>
      </tr><tr><td>
          <p><code><a href="dot-protein_boxplot.html">.protein_boxplot()</a></code> </p>
        </td>
        <td><p>Boxplots of phosphor-proteomics data across clinic stages</p></td>
      </tr></tbody></table></div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Su Wu.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.0.6.</p>
</div>

      </footer></div>

  


  

  </body></html>

